MetaMask/metamask-extension
# fix(hardware-wallets): update @metamask/eth-trezor-keyring

## **Description**

- Update `@metamask/eth-trezor-keyring` to latest
- Update `@trezor/connect-web` to latest

Contains multiple fixes and improvements for Trezor users.

Removes dependency on broken `protobufjs@6.11.3`.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23164?quickstart=1)

## **Related issues**

- Fixes: #10896
- https://github.com/MetaMask/eth-trezor-keyring/issues/187

## **Manual testing steps**
(you need a physical Trezor device or [you can use a simulator here](https://github.com/trezor/trezor-user-env))

1. Go to [MM test dapp](https://metamask.github.io/test-dapp)
2. Connect with a Trezor account
3. check Personal Sign, Sign Typed Data V4 or Sign Permit and see that features are still working.

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
